### PR TITLE
create(docs) create install page vue

### PR DESCRIPTION
**Pull request by Sicorschi-Tudor**
https://github.com/bagrinsergiu/blox-editor/issues/23322

### CODE REVIEW

File: packages/tsconfig/vue-library.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "https://json.schemastore.org/tsconfig",
+  "display": "Vue Library",
+  "extends": "@vue/tsconfig/tsconfig.json",
+  "compilerOptions": {
+    "declaration": true,
+    "declarationMap": false,
+    "ignoreDeprecations": "5.0"
```
Comment (maxval1) [inline packages/tsconfig/vue-library.json:8]: sa facut update la ts 5 vezi daca mai trebue `ignoreDeprecations`

**Review by maxval1 — CHANGES_REQUESTED** [review]
1. de rezolvat conflictele
2. de adaugat in `torbo.json` @brizy/docs-vue#dev
3. de adaugat in package.json din documentation-vue `prettier-check` task vezi in docs de la react
4.  @CristianSpatari a facut un fix la docs din react, verifica te rog sa fie si in docs din vue

File: packages/tsconfig/vue-library.json (hunk 1 above)
Comment (Sicorschi-Tudor) [reply packages/tsconfig/vue-library.json:8]: Screenshot from 2023-07-03 10-48-49


    "ignoreDeprecations": "5.0" - oricum trebuie

**Review by maxval1 — CHANGES_REQUESTED** [review]
de lamurit cu https://github.com/vuejs/tsconfig

**Pull request merged**